src/elarm_event.erl

```diff
@@ -50,19 +50,19 @@
 init(_Opts) ->
     {ok, #evt_state{}}.
 
--spec subscribe(pid()|atom(), sub_filter(), #evt_state{}) ->
-          {{ok,reference()},#evt_state{}}.
+-spec subscribe(pid() | atom(), sub_filter(), #evt_state{}) ->
```

Comment:
The spec in line 53 uses a record, please define a type for the record and use that instead.